src/components/Note/Note.tsx

```diff
@@ -28,22 +28,47 @@ export const Note = (props: NoteProps) => {
 
   const note = useAppSelector((state) => state.notes.find((n) => n.id === props.noteId), isEqual);
   const isStack = useAppSelector((state) => state.notes.filter((n) => n.position.stack === props.noteId).length > 0);
+  const noteChildren = useAppSelector((state) => state.notes.filter((n) => n.position.stack === props.noteId));
```

Comment:
Since you don't need the whole notes but only the authors, you should already map the notes inside the selector. Through that we can prevent the note from unnecessary rerenderings when the text of child notes has been changed.

Reply:
So basically do that mapping from line 41 already inside the `useAppSelector` please

Reply:
I have changed it so all filtering and mapping of `childrenNoteAuthors` is done in the selector, is this what you meant?